app/grants/views.py

```diff
@@ -854,7 +862,12 @@ def grants_by_grant_type(request, grant_type):
     _grants = None
     bg, mid_back, bottom_back, bg_size, bg_color = get_bg(grant_type)
     show_past_clr = False
+    all_grant_types = GrantType.objects.all()
 
+    all_styles = {}
+    for _gtype in all_grant_types:
+        bg, mid_back, bottom_back, bg_size, bg_color = get_bg(_gtype)
+        all_styles[_gtype.name] = dict(bg=bg, mid_back=mid_back, bottom_back=bottom_back, bg_size=bg_size, bg_color=bg_color)
```

Comment:
missing inline_css is a style block added on each page to setup each banner special styles needs.